src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,153 @@
+ `<datatype>` directory (for example `func` when combined with fMRI, or
+ `beh` when combined with behavioral measures).
+
+In the template filenames, the `<matches>` part corresponds to task filename
+before the suffix. For example, when fMRI is considered the main modality with
+files such as `sub-01_ses-1_task-pRF_run-1_bold.nii.gz`, `<matches>` would
+then correspond to `sub-01_ses-1_task-pRF_run-1`.
+
+### Tabular file (`*_eyetrack.tsv.gz`)
+
+The eye-tracking data must be saved in `.tsv.gz` format.
+The tabular files consist of one row per sample and a set of REQUIRED and
+OPTIONAL columns.
+
+<!--
+This block generates a columns table.
+The definitions of these fields can be found in
+  src/schema/rules/tabular_data/eyetrack.yaml
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_columns_table("eyetrack.EyeTracking") }}
+
+Throughout BIDS you can indicate missing values with `n/a` (for "not
+available").
+
+We encourage users to put the eye-tracking raw data within the
+`/sourcedata` directory (for example put eye-tracker proprietary files before
+conversion to `tsv.gz` format in the sourcedata directory).
+
+### Sidecar JSON document (`*_eyetrack.json`)
+
+<!-- This block generates a metadata table.
+These tables are defined in
+ src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+ src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("eyetrack.EyeTrackingRequired") }}
+
+Note that `ScreenSize`, `ScreenResolution` and `ScreenDistance` are REQUIRED as
+they are considered essential in eye-tracking data analysis.
+
+**Specific eye-tracking fields SHOULD be present:**
+
+<!-- This block generates a metadata table.
+These tables are defined in
+ src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+ src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("eyetrack.EyeTrackingRecommended") }}
+
+**Specific eye-tracking fields MUST be present:**
+
+### Example sidecar JSON document
+
+```JSON
+{
+"Manufacturer": "SR-Research",
+"ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
+"SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
+"SamplingFrequency": 1000,
+"SampleCoordinateUnits": "pixel",
+"SampleCoordinateSystem": "gaze-on-screen",
+"EnvironmentCoordinates": "top-left",
+"RecordedEye": "Both",
+"ScreenSize": [38.6, 29],
+"ScreenResolution": [1024, 768],
+"ScreenDistance": 60,
+"ScreenAOIDefinition": ["square",
+                        ["x_start", "x_stop", "y_start", "y_stop"]]
```

Comment:
change into actual values

Reply:
Yes, you are right, I changed it to `"ScreenAOIDefinition": ["square",[100, 150, 300, 350]]`
but what do you think should be the "datatype".
Now we used "object of objects"...